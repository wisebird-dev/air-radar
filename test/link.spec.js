import React from 'react';
import {shallow} from 'enzyme';

import Link from '../src/components/link';

test('it works', () => {
	const wrapper = shallow(<Link/>);
	expect(wrapper).toMatchSnapshot();
});
