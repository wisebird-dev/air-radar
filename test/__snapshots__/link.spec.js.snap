// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
    color: #fafff7;
    box-sizing: border-box;
    text-decoration: underline;
    user-select: none;
    
    &:hover {
    text-decoration: none;
    background: #fdfdfd;
    color: #1b1b1b
  }
",
        ],
      },
      "displayName": "styled.a",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "a",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
